test/scripts/install-provisioner-working.sh

```diff
@@ -49,6 +47,8 @@ ${KUBECTL} apply -f <(cat examples/gateway-provisioner/03-gateway-provisioner.ya
     yq eval '.spec.template.spec.containers[0].imagePullPolicy = "IfNotPresent"' - | \
     yq eval '.spec.template.spec.containers[0].args += "--contour-image="+env(CONTOUR_IMG)' -)
 
+${KUBECTL} patch deployment -n projectcontour contour-gateway-provisioner --patch '{"spec": {"template": {"metadata": {"annotations": {"timestamp": "'$(date +%s)'"}}}}}'
```

Comment:
There could be small comment to explain the purpose: patching the deployment with timestamp will trigger pod restart.